BaseApi/V1/Gateways/DayCarePackageGateways/DayCarePackageGateway.cs

```diff
@@ -12,6 +16,7 @@ public DayCarePackageGateway(DatabaseContext dbContext)
         {
             _dbContext = dbContext;
         }
+
         public async Task<Guid> CreateDayCarePackage(Infrastructure.Entities.DayCarePackage dayCarePackage)
```

Comment:
It would be beneficial to change Create to Upsert (update+insert), we have an example available for this, it will allow us to consolidate the code and not repeat field updates